src/EventSubscriber/AddInstantPurgeHeaders.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Drupal\fastly\EventSubscriber;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\fastly\Form\FastlySettingsForm;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Generates additional headers for instant purge
+ */
+class AddInstantPurgeHeaders implements EventSubscriberInterface {
```

Comment:
Noting that 4 spaces is being used throughout the doc instead or 2. We should make sure to follow Drupal coding standards.